### PR TITLE
add button css

### PR DESCRIPTION
**Pull request by fjnelson**
added div container for buttons in index
added container align in css
not working

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -36,4 +36,7 @@ h1
 }
 .myButton:active {
     background: #144E75; 
+}
+.link-container {
+    align-items: center;
```
Comment (nelsonsbrian) [inline style.css:41]: Think `align-items` only works if you have `display: flex` as well?

https://www.w3schools.com/CSSref/css3_pr_align-items.asp

Comment (fjnelson) [reply style.css:41]: i'll have to play around with containers later, at least i am assuming i need to group the buttons in a container to move them around?

**Comment by fjnelson** [comment]
closing unresolved

File: style.css (hunk 1 above)
Comment (nelsonsbrian) [reply style.css:41]: Yea whatever you do to the container will move the 3 links inside. Easy to stack buttons horizontally, vertically, and adjust spacing. Nothing inherently special about a 'container' is just a name for an empty div element that has other elements as children.

**Pull request closed without merging**